src/main/java/net/snowflake/client/core/SessionUtilKeyPair.java

```diff
@@ -178,7 +183,13 @@ public String issueJwtToken() throws SFException {
     } catch (JOSEException e) {
       throw new SFException(e, ErrorCode.FAILED_TO_GENERATE_JWT);
     }
-
+    // Log the contents of the token, displaying expiration and issue time in epoch time
```

Comment:
shows up in the logs as:
`2021-04-13 22:00:43.590 n.s.c.core.SessionUtilKeyPair FINE issueJwtToken:188 - JWT:
{
iss: TESTACCOUNT.SNOWMAN.SHA256:tqTTSFoHg1prjjzurTGBaroVoGifEugI+tLVVAOlkfM=
sub: TESTACCOUNT.SNOWMAN
iat: 1618351243
exp: 1618351303
}`